source/clear-linux/get-started/compatibility-check.rst

```diff
@@ -34,31 +33,24 @@ On a system that is currently running a Linux operating system, follow the instr
 
          $ ./clear-linux-check-config.sh container
 
-   The script will print a list of test results similar to the output below.
-   All items should return a *SUCCESS* status, thus indicating the target
-   system fully supports running |CL|.
+   The script will print a list of test results similar to the output below.  
+   All items should return a *SUCCESS* status.  This example indicates the host's prcessor and EFI firmware supports running |CL|.
 
    .. code-block:: console
 
       Checking if host is capable of running |CL|\* OS for Intel®
       Architecture
 
-      SUCCESS:  Intel CPU
+      SUCCESS: Intel CPU
 
       SUCCESS: 64-bit CPU (lm)
 
-      SUCCESS:  Streaming SIMD Extension v4.1 (sse4_1)
+      SUCCESS: Streaming SIMD Extension v4.1 (sse4_1)
 
       SUCCESS: EFI Firmware
 
-.. note::
-
-   Only a system running a Linux distribution can run the compatibility
-   check. There are two alternative options:
-
-   * Install and run a Linux distribution directly on your system.
    * Run a :ref:`live-image` from a USB drive (success does not guarantee your
      system is 100% compatible).
      
  .. _clear-linux-check-config.sh file: https://download.clearlinux.org/current/clear-linux-check-config.sh
-     
+ .. _live image: https://clearlinux.org/documentation/clear-linux/get-started/live-image.html    
```

Comment:
Will remove this so that the url doesn't become out of date and use the :ref: role.